src/Tools/CompilerPerfTests/CompilerPerfTests.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Need to avoid the new MSBuild format for now. As stands this won't build to our standard Binaries folder, and instead directly into the source tree.

Please use the existing patterns here. This will get converted to the new SDK when I move the entire repo.

Reply:
@jaredpar I don't understand what the limitation is. Since this is a one-off tool, why does it matter where it builds to? It's not part of any solution or CI build.